projects/EditionsIgnite/app/app.tsx

```diff
@@ -65,7 +65,7 @@ export class App extends React.Component<{}, AppState> {
     return (
       <Provider rootStore={rootStore} navigationStore={rootStore.navigationStore} {...otherStores}>
         <BackButtonHandler canExit={this.canExit}>
-          <StatefulNavigator />
+          <RootNavigator />
```

Comment:
This simplifies things a little, but we need to make a decision on whether to remove statefulness entirely. I'm not sure exactly what we're losing at the moment

Reply:
given that this is uncharted territory i'm pretty happy to let react manage the state at a component level for us (the happy path) until we either run into issues or there's things that we just know for sure we wanna use something like redux/mobx for

Reply:
what is the difference? happy to learn tomorrow in person!